### ISSUE TITLE
[Bug][Pimcore 10.6] Creating static routes not allowed 

### ISSUE THREAD

**Issue by fashxp** [pimcore/pimcore#15100]
### Pimcore version

10.6

### Steps to reproduce

- Install Pimcore 10.6
- Try to create static routes 

### Expected / actual behavior

![image](https://user-images.githubusercontent.com/8792145/236034133-5126c24d-8d19-41d4-ba70-2fbd4a36ac68.png)

Add button is deactivated, regardless of configuration write target and environment

**Comment by dvesh3** [comment]
works for me on fresh installation of 10.6:
<img width="250" alt="image" src="https://github.com/pimcore/pimcore/assets/5137917/606f050b-602e-496c-952e-1bb752eb8932">

even works with different write locations yaml config and setting it disabled.

**Comment by fashxp** [comment]
In prod env too?

**Comment by brusch** [comment]
It depends on the kernel's debug flag (e.g. `APP_DEBUG=1`), the env is not relevant 😉

**Comment by fashxp** [comment]
@dvesh3 following config should allow creation of static routes in settings-store, but it doesn't for me

```
APP_ENV=prod
APP_DEBUG=false
PIMCORE_DEV_MODE=false

PIMCORE_WRITE_TARGET_STATICROUTES=settings-store
```

**Comment by dvesh3** [comment]
yes, I didn't check on prod with debug false before. will do now! thanks for sharing detailed steps.

**Comment by dvesh3** [comment]
Fixed by #15158
